public/app/api/clients/folder/v1beta1/hooks.ts

```diff
@@ -320,6 +323,38 @@ export function useCreateFolder() {
   return [createFolderAppPlatform, result] as const;
 }
 
+export function useUpdateFolder() {
+  const [updateFolder, result] = useReplaceFolderMutation();
```

Comment:
Would probably just add a comment that replace and update are the same thing, otherwise this is a bit confusing.

Reply:
Based on conversations in #app-platform channel, I'll consolidate these to both use `replace` (perhaps in a follow up PR to tidy the hooks and tests up)